alvr/server_openvr/cpp/alvr_server/Controller.h

```diff
@@ -54,6 +54,9 @@ class Controller : public TrackedDevice, public vr::ITrackedDeviceServerDriver {
     vr::EVRSkeletalTrackingLevel m_skeletonLevel;
 
     vr::DriverPose_t m_pose;
+    vr::DriverPose_t m_lastPose;
+    uint64_t m_poseTargetTimestampNs;
+    uint64_t m_lastPoseTargetTimestampNs;
```

Comment:
m_lastPose and m_lastPoseTargetTimestampNs are unused

Reply:
oh oops I thought I removed that